client-app/src/css/AddProgramPage.css

```diff
@@ -1,39 +1,72 @@
 .container {
```

Comment:
Once again, like last week, I overall think this form looks really good. It's nice, short, and gets to the point. Good job!